integrations/shuffle.py

```diff
@@ -41,57 +42,60 @@
 # </integration>
 
 # Global vars
-debug_enabled   = False
 pwd             = os.path.dirname(os.path.dirname(os.path.realpath(__file__)))
 json_alert      = {}
 SKIP_RULE_IDS   = ["87924", "87900", "87901", "87902", "87903", "87904", "86001", "86002", "86003", "87932",
                  "80710", "87929", "87928", "5710"]
 
-# Log path
-LOG_FILE        = f'{pwd}/logs/integrations.log'
+# Logger
+LOG_FILE    = f'{pwd}/logs/integrations.log'
+logger      = logging.getLogger("shuffle")
 
 # Constants
 ALERT_INDEX     = 1
 WEBHOOK_INDEX   = 3
 
 
 def main(args):
-    global debug_enabled
     try:
         # Read arguments
-        bad_arguments: bool = False
+        invalid_arguments: bool = False
         if len(args) >= 4:
             msg = '{0} {1} {2} {3} {4}'.format(
                 args[1],
                 args[2],
                 args[3],
-                args[4] if len(sys.argv) > 4 else '',
+                args[4].upper() if len(args) > 4 else 'INFO',
                 args[5] if len(sys.argv) > 5 else ''
             )
-            debug_enabled = (len(args) > 4 and args[4] == 'debug')
         else:
-            msg = '# ERROR: Wrong arguments'
-            bad_arguments = True
+            invalid_arguments = True
 
-        # Logging the call
-        with open(LOG_FILE, "a") as f:
-            f.write(msg + '\n')
+        if invalid_arguments:
+            print_help_msg()
+            sys.exit(ERR_INVALID_ARGUMENTS)
```

Comment:
Same as for slack, I would move this to the `else` clause and remove the `invalid_arguments` variable.